File: PRAK602-2310817210026-MUHAMMADPUTRAAZKYALFANI.py
jumlah_ruangan = int(input("Masukkan jumlah ruangan: "))
zetsu_putih = list(map(int, input(f"Masukkan jumlah zetsu putih untuk setiap ruangan: ").split()))

for i in range(jumlah_ruangan):
    zetsu_putih[i] *= (i + 1)

print("Jumlah zetsu putih setelah membelah diri pada setiap ruangan:")
print(" ".join(map(str, zetsu_putih)))

File: PRAK602-2310817210026-MUHAMMADPUTRAAZKYALFANI.c
#include <stdio.h>

int main() {
    int jumlahRuangan;
    printf("jumlah ruangan: ");
    scanf("%d", &jumlahRuangan);

    int zetsuPutih[jumlahRuangan];

    printf("Masukkan jumlah zetsu putih untuk setiap ruangan: ");
    for (int i = 0; i < jumlahRuangan; i++) {
        scanf("%d", &zetsuPutih[i]);
    }

    for (int i = 0; i < jumlahRuangan; i++) {
        zetsuPutih[i] *= (i + 1);
    }

    printf("Jumlah zetsu putih setelah membelah diri pada setiap ruangan:\n");
    for (int i = 0; i < jumlahRuangan; i++) {
        printf("%d ", zetsuPutih[i]);
    }

    return 0;
}
